src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BucketHelper.scala

```diff
@@ -102,11 +131,11 @@ class BucketHelper(dataprocConfig: DataprocConfig,
     }
   }
 
-  private def userEntity(email: WorkbenchEmail) = {
-    GcsEntity(email, User)
-  }
+  private def userEntity(email: WorkbenchEmail) = EmailGcsEntity(User, email)
+  private def groupEntity(email: WorkbenchEmail) = EmailGcsEntity(Group, email)
+}
 
-  private def groupEntity(email: WorkbenchEmail) = {
-    GcsEntity(email, Group)
-  }
+object BucketHelper {
```

Comment:
As a convention we've been putting `LeoExceptions` at the top of the file, _not_ inside a companion object.